server/consumer.go

```diff
@@ -4375,3 +4583,30 @@ func (o *consumer) processStreamSignal(_ *subscription, _ *client, _ *Account, s
 		o.signalNewMessages()
 	}
 }
+
+// Used to compare if two multiple filtered subject lists are equal.
+func subjectSliceEqual(slice1 []string, slice2 []string) bool {
```

Comment:
reflect.DeepEqual()?

Reply:
DeepEqual would return `false` if the slices contain the same subjects, but passed in different order.

I so no reason to not treat them as equal.

Reply:
Agree they should be equal, I forgot that they needed to be in same order.